### ISSUE TITLE
Rating is missing keyboard focus

### ISSUE THREAD

**Issue by GarthDB** [adobe/spectrum-css#249]
@NateBaldwinDesign is there a design spec for the keyboard focus for Rating?

**Comment by NateBaldwinDesign** [comment]
There is a design spec, and it was implemented in the prototype @lazd built for this. Larry may have insight into this.

**Comment by NateBaldwinDesign** [comment]
![image](https://user-images.githubusercontent.com/13972198/63539342-17557500-c4d7-11e9-965d-0953d96b0af8.png)

**Comment by NateBaldwinDesign** [comment]
On keyboard focus, the entire control receives the focus ring. The user can arrow left/right to change the value

**Comment by lazd** [comment]
@GarthDB the CSS contains an `is-focused` class that applies the focus style. This is not shown in the docs because, in reality, the `<input>` within the rating gets the focus, though we could add something to `enhancements.js` to enable it.

**Comment by GarthDB** [comment]
It would be nice to have some way to preview the focus state.

**Comment by thnat1234** [comment]
Hi, 
I am a first-time contributor and would like to work on this issue. Can I start working on this issue, please?

**Comment by lazd** [comment]
@thnat1234 absolutely. Go ahead and send a PR adding an example with the `.is-focused` class. Thanks!
